tests/unit/test_meta_model_repurposer.py

```diff
@@ -179,6 +179,8 @@ def _test_save_load_repurposed_model(self, mock_model_handler, save_source_model
         source_model = mx.module.Module.load('tests/data/testnetv1', 0, label_names=['softmaxoutput1_label'],
                                              data_names=('data',))
         repurposer = self.repurposer_class(source_model, self.source_model_layers)
+        if self.repurposer_class == BnnRepurposer:
+            repurposer = self.get_repurposer(source_model, self.source_model_layers)
```

Comment:
Consider creating BnnRepurposer object here directly. Calling self.get_repurposer which is not declared in base class doesn't seem right.